docs/data/base/components/input/input.md

```diff
@@ -0,0 +1,41 @@
+---
+product: base
+title: React Input unstyled component and hook
+components: InputUnstyled
+githubLabel: 'component: input'
+packageName: '@mui/base'
+---
+
+# Input
+
+<p class="description">Inputs let users enter and edit text.</p>
+
```

Comment:
Should we have a Features list here as well? Just as we did on the Modal doc.
Also, should we list the Material UI components that use Input as a lower-level construct?

Reply:
It's a bit tricky here to make a list, because currently none of the Material UI components use the unstyled input. On the other hand, the `ModalUnstyled` is used in the Material UI components (when we started with the unstyled components, we were actually using them in the Material UI components, but after some time, we decided not to, so that we don't block the progress with v5). I would wait for v6 and the components to actually be used before listing them here.

Reply:
Okay! What about the features list?!

Reply:
That's a good point, I will add it.

Reply:
Done.